jenkins/agent-ctl.sh

```diff
@@ -309,7 +309,7 @@ which docker || curl -sSL https://get.docker.com/ | sh
 id jenkins || sudo useradd jenkins -m
 sudo usermod -aG docker jenkins
 
-# Downgrade to 1.9.1
+# Downgrade docker to 1.11.2, the version on COS
 # https://github.com/kubernetes/kubernetes/issues/21451
```

Comment:
not especially, but it gives at least some history as to why we're pinning to a version instead of using latest.